README.markdown

```diff
@@ -28,7 +28,7 @@ Installation
 
 Install Redigo using the "go get" command:
 
-    go get github.com/gomodule/redigo/redis
+    go get -u github.com/gomodule/redigo/redis/v3
```

Comment:
```suggestion
    go get github.com/gomodule/redigo/redis/v3
```
I believe the recommended way is not to use `-u` with `go mod`

Reply:
[Yup](https://github.com/golang/go/wiki/Modules#updating-pre-existing-install-instructions), OK.